src/app/App.vue

```diff
@@ -99,8 +104,28 @@ const route = useRoute()
  */
 const routeKey = computed(() => route.path)
 const isWizard = computed(() => route.meta.isWizard === true)
-const shouldShowAppError = computed(() => store.getters.shouldShowAppError)
```

Comment:
Again a fairly local piece of state, so I switched this out for a local piece of state.